pkg/controller/autoscaler/horizontalpodautoscaler_controller.go

```diff
@@ -17,21 +17,16 @@ limitations under the License.
 package autoscalercontroller
```

Comment:
Thanks for the suggestions, I'll work on it in a separate PR, as soon as this PR is merged.